Dockerfile

```diff
@@ -1,21 +1,21 @@
-FROM node:14-alpine AS BUILD
+FROM node:16-alpine AS BUILD
 
 # git is needed to install Half-Shot/slackdown
 RUN apk add git
 WORKDIR /src
 
 COPY package.json package-lock.json /src/
-RUN npm install
+RUN npm install --ignore-scripts
```

Comment:
Isn't this a good time to switch it to `npm ci`, so that we actually respect what's in package-lock?

Reply:
This is an excellent time for that, how could I forget.